docs/tooling/catwalk.md

```diff
@@ -14,5 +14,19 @@ within.
 
 You can use catwalk in the following ways:
 
-* provide a link to an engine web socket at [catwalk](https://catwalk.core.qlik.com/) to view your Qlik Core app
-* clone the [catwalk repo](https://github.com/qlik-oss/catwalk) and run the tool locally
\ No newline at end of file
+* provide a link to an engine web socket at [catwalk](http://catwalk.core.qlik.com/)
+* open an application from corectl by `corectl catwalk --app my-app.qvf`
+* clone the [catwalk repo](https://github.com/qlik-oss/catwalk) and run the tool locally
+
+If no web socket configuration is provided, catwalk defaults to localhost:9076.
+
+!!! Note
+    The Qlik application's content is transferred directly from the engine to the web browser through a web socket. I.e.
+    no data is sent to Qlik or any 3:rd party. Beside this Qlik collects anonymous usage data such as browser version
+    and catwalk exceptions.
+
+## Best practices and techniques
+
+Catwalk provides an alternative view of the Qlik application's data model compared to existing products but best
```

Comment:
This sentence is a bit strange when reading. Maybe change to `Catwalk provides an alternative view of the Qlik application's data model compared to existing products. However best practices and data modelling techniques are still the same and can be found on the...`